##this message will contain the measured velocity of the create as determined by wheel encoders

float64 leftVm
float64 rightVm
float64 dt
